src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.LocalFunctions.cs

```diff
@@ -133,7 +133,15 @@ public override BoundNode VisitLocalFunctionStatement(BoundLocalFunctionStatemen
             var oldReads = usages.ReadVars;
             usages.ReadVars = BitVector.Empty;
 
-            if (!localFunc.WasCompilerGenerated) EnterParameters(localFuncSymbol.Parameters);
+            if (!localFunc.WasCompilerGenerated)
+            {
+                EnterParameters(localFuncSymbol.Parameters);
+
+                foreach (var parameter in localFuncSymbol.Parameters)
+                {
+                    _variableUsePass.Visit(parameter.ExplicitDefaultExpression);
+                }
```

Comment:
Perhaps call `_variableUsePass.Visit()` in `EnterParameter` rather than having separate loop here.